test/auto/tests/tests.xml

```diff
@@ -5,15 +5,13 @@
       <description>Ofono test set with manual, automatic and semi-automatic case.</description>
       <pre_steps>
         <step>/usr/sbin/mcetool --set-never-blank=disabled --set-demo-mode=off --block=1 --set-never-blank=enabled --unblank-screen --block=1 --set-tklock-mode=unlocked</step>
-        <step>/usr/sbin/run-blts-root /bin/mv /var/lib/environment/ofono/noplugin.conf /var/lib/environment/ofono/noplugin.conf_LIBQOFONO_TESTS_DISABLED || true</step>
-        <step>/usr/sbin/run-blts-root /bin/sh -c "/bin/echo OFONO_ARGS=--noplugin=ril > /var/lib/environment/ofono/LIBQOFONO_TESTS_noplugin.conf" || true</step>
+        <step>/usr/sbin/run-blts-root /bin/sh -c "/bin/echo OFONO_ARGS=--noplugin=ril > /var/lib/environment/ofono/xxx_LIBQOFONO_TESTS_noplugin.conf" || true</step>
```

Comment:
Guess "xxx" for ensuring it's the last conf file read. Could consider adding some note.

Reply:
Exactly. I added a comment.